src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,39 +81,47 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-	String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
- 		
-	String textReplaced = replaceUserText(userText);   
-	
-	Message message = new Message(user, textReplaced);
-	datastore.storeMessage(message);
+	//String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+
+	String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+	  TextProcessor processor = BBProcessorFactory.getInstance().create();
+	  String input= processor.process( text ) ;
```

Comment:
Add whitespace on both sides of the `=` character.
Don't add whitespace after `(`.
Please review Java style guide: https://google.github.io/styleguide/javaguide.html#s4-formatting